src/test/c_api_tests.cpp

```diff
@@ -236,10 +237,79 @@ TEST_F(CapiInferencePreparationTest, Basic) {
     //////////////////
     //  INFERENCE
     //////////////////
+    // we need to prepare server now TODO
+    // remove when C-API start implemented
+    std::string port = "9000";
+    randomizePort(port);
+    char* argv[] = {
+        (char*)"OpenVINO Model Server",
+        (char*)"--model_name",
+        (char*)"dummy",
+        (char*)"--model_path",
+        (char*)"/ovms/src/test/dummy",
+        (char*)"--port",
+        (char*)port.c_str(),
+        (char*)"--log_level",
+        (char*)"DEBUG",
+        nullptr};
+
+    ovms::Server& server = ovms::Server::instance();
+    std::thread t([&argv, &server]() {
+        ASSERT_EQ(EXIT_SUCCESS, server.start(9, argv));
+    });
+    auto start = std::chrono::high_resolution_clock::now();
+    while ((ovms::Server::instance().getModuleState("GRPCServerModule") != ovms::ModuleState::INITIALIZED) &&
```

Comment:
This while loop could be removed since c-api server start blocks until all models are loaded

Reply:
The test would not need sleeps as well

Reply:
This should be already resolved.